torch/csrc/jit/tensorexpr/ir_simplifier.cpp

```diff
@@ -2822,6 +2872,49 @@ bool exprEquals(ExprPtr A, ExprPtr B) {
   }
 }
 
+ExprPtr IRSimplifier::simplify(ExprPtr e) {
+  GRAPH_DEBUG("(Simplifier) Original: ", std::to_string(e));
```

Comment:
Thanks for adding these. I've been wanting to add this to quickly check if simplifier is correct (given the complicated bugs we have in this).